src/components/common/ForeignJurisdiction.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-card flat class="pt-6 px-4">
```

Comment:
You should assign an id or a class to name this component. (Sort of a general practise for all components, if not used for styling then maybe used for testing.)